### ISSUE TITLE
None of the twitch anti ad solutions work. Ads keep playing in full quality on every twitch stream. (Firefox)

### ISSUE THREAD

**Issue by Noobeley** [pixeltris/TwitchAdSolutions#305]
I've enabled and disabled every single one of these solutions, even ttv lol pro v2 on maximum proxy usage, but no matter what, ads still play.

Can someone please write a script that works when none of the other solutions work, that simply detects the words indicating that an ad is playing somewhere on the screen, and during that, mute the audio and black out the video player?

**Comment by pixeltris** [comment]
> that simply detects the words indicating that an ad is playing somewhere on the screen, and during that, mute the audio and black out the video player?

`Alternate Player for Twitch.tv` ([chrome](https://chrome.google.com/webstore/detail/alternate-player-for-twit/bhplkbgoehhhddaoolmakpocnenplmhf) / [firefox](https://addons.mozilla.org/en-US/firefox/addon/twitch_5/)) does exactly that. It's linked in the README. This is what it looks like:

![twitch5](https://github.com/user-attachments/assets/36f696b9-fc44-4caf-af90-b1025b64a077)

**Comment by Noobeley** [comment]
No, alternate twitch player is a stopgap solutions because it completely overhauls the UI and forces a different player and chat. I just need the bare minimum, not to see ads while they are thinking they're still being played.

I still want to use the normal twitch site with moderation tools, list of streamers live on the left, access to streamer's bio, and stream addons and overlays.
